### DISCUSSION TITLE
Package removal from Registry

### DISCUSSION THREAD

**Discussion by streambinder** [community/community#27089] [Actions]
<p>Hi. I am trying to manually and forcefully remove a container image associated to a repository of mine from the old registry <code>docker.pkg.github.com</code>.<br>
As you can check by going to the <a href="https://github.com/streambinder/streambinder/packages/" rel="noopener nofollow ugc">repository I am talking about</a>, it is releasing two images, a single-versioned one from the aforementioned old registry and the other one, actively maintained and updated from the newly introduced <code>ghcr.io</code> registry.<br>
I know there’s a <a href="https://docs.github.com/en/free-pro-team@latest/packages/guides/deleting-a-container-image" rel="noopener nofollow ugc">documentation page</a> which is pretending the task to be extremely easy, but in my case there’s no delete button for either an image or a specific version (and this only applies to images from the former registry, as I can perfectly see the buttons for images/versions from the <code>ghcr.io</code> registry).<br>
I also tried using bunch of cURL requests via CLI, but the functionality is apparently not supported via CLI: <code>{"message":"Package deletion from the command line is disabled in GitHub Package Registry. For further details please visit https://help.github.com/en/packages/publishing-and-managing-packages/deleting-a-package."</code>.<br>
Any help so far?</p>

**Comment by whitneyimura** [comment, marked as answer]
<p>Hi <a class="mention" href="/streambinder">@streambinder</a> – We currently do not support public package deletion for Docker packages (on <a href="http://docker.pkg.github.com" rel="noopener nofollow ugc">docker.pkg.github.com</a>), but we are working on adding that support which should be coming very shortly.</p>

**Comment by streambinder** [comment]
<p>I can confirm the feature has been added: I managed to remove the package straight from GitHub page.</p>
